k8s-operator/apis/v1alpha1/types_tsdnsconfig.go

```diff
@@ -75,6 +75,10 @@ type DNSConfigSpec struct {
 	// Tailscale Ingresses. The operator will always deploy this nameserver
 	// when a DNSConfig is applied.
 	Nameserver *Nameserver `json:"nameserver"`
+	// Domain is the domain for which DNS entries will be resolved.
+	// Defaults to ts.net if empty.
```

Comment:
The flag in the nameserver defaults to `ts.net` but here it does not so this comment is not super accurate. Maybe a kubebuilder default here would make sense?

Reply:
I was thinking about this, but I didn't see any kubebuilder default annotation used in any other type file. That is why I went without. I could change the comment to

```
If left empty, the default of the k8s-nameserver will be used.
```
?

Reply:
Furthermore the domain is currently defined by the control-server and by the k8s-nameserver, but the k8s-operator is quite domain agnostic. The "ts.net" domain is only used in test files.

Reply:
> If left empty, the default of the k8s-nameserver will be used.
Fine with me

Reply:
I changed it.